setup.py

```diff
@@ -28,9 +28,8 @@
     url=about['__url__'],
     license=about['__license__'],
     # Exclude tests from built/installed package.
-    packages=setuptools.find_packages(
-        exclude=['tests', 'tests.*', '*.tests', '*.tests.*']
-    ),
+    packages=setuptools.find_packages("src", exclude=["tests", "tests.*", "*.tests", "*.tests.*"]),
```

Comment:
Is the exclude required as you moved tests out of `src`?

Reply:
I'm not sure, i've copied this from [DRF-reactive](https://github.com/genialis/django-rest-framework-reactive/blob/master/setup.py) that has the same structure :-)

Reply:
Thanks for this comment. If i remove it all works fine, so i'll just leave it without exclude.